p2p/src/peer_manager/peerdb/address_data/mod.rs

```diff
@@ -47,6 +47,10 @@ pub enum AddressState {
 
         /// Next time connect to the peer
         next_connect_after: Duration,
+
+        /// It's a flag for outbound peers that have been manually disconnected
```

Comment:
Disconnect is done by requesting to do it. This flag ensures that p2p will not try to reconnect to the target node.